geonode/ogpsearch/utils.py

```diff
@@ -49,10 +50,7 @@ def get_domain(url):
         hostname = urlParts.hostname
         if hostname == "localhost":
             return "Harvard" # assumption
-        domainParts = hostname.split(".")
-        if len(domainParts) == 1:
-            return domainParts[0]
-        return domainParts[-2].capitalize()
+        return hostname
```

Comment:
Do you know why it was splitting the other parts of the domain?

I would suggest to search where get_domain is being used in the codebase to ensure it does not cause trouble.